hack/run-ci-e2e-test.sh

```diff
@@ -10,9 +10,50 @@ NODE_COUNT=""
 SKIP_NODE_DELETION=""
 KEY_PAIR_NAME=""
 
-
 export CGO_ENABLED=0
 
+get_WMCO_logs() {
+  oc logs -l name=windows-machine-config-operator -n windows-machine-config-operator
+}
+
+# This function runs operator-sdk run --olm/cleanup depending on the given parameters
+# Parameters:
+# 1: command to run [run/cleanup]
+# 2: path to the operator-sdk binary to use
+# 3: path to the directory holding the operator manifests
+OSDK_WMCO_management() {
+  if [ $# != 3 ]; then
+    echo incorrect parameter count for OSDK_WMCO_management
+    return 1
+  fi
+  if [[ "$1" != "run" && "$1" != "cleanup" ]]; then
+    echo $1 does not match either run or cleanup
+    return 1
+  fi
+
+  # Currently this fails even on successes, adding this check to ignore the failure
+  # https://github.com/operator-framework/operator-sdk/issues/2938
+  if ! $2 $1 --olm --olm-namespace openshift-operator-lifecycle-manager --operator-namespace windows-machine-config-operator --operator-version 0.0.0 --manifests $3/windows-machine-config-operator; then
```

Comment:
nit: I'd add local variables to store $2 and $1 to improve readability

Reply:
That will actually be quite useful.

Reply:
I thought the function was short enough to avoid doing that. I dislike setting variables in bash functions unless necessary as they are not scoped. If its unreadable I can change it.

Reply:
You can use `local` to scope them locally to a function.